test/blocks/brick/mocks/body.html

```diff
@@ -40,7 +40,7 @@ <h3 id="heading-xxl-4455-lorem-ipsum">Heading XXL 44/55 Lorem ipsum.</h3>
         </div>
       </div>
     </div>
-    <div class="brick">
+    <div class="brick m-rounded-corners">
```

Comment:
I see the mock file has been updated for rounded corners but don't see test.js file with test case added as part of this PR. Could you please confirm on that.